datalab/settings.py

```diff
@@ -130,6 +130,11 @@ def get_list_from_env(variable, default=None):
 # AdminMiddleware is enabled.  The default value is 'default'.
 DRAMATIQ_TASKS_DATABASE = 'default'
 
+# AWS S3 Bitbucket
+DATALAB_OPERATION_BUCKET = os.getenv('DATALAB_OPERATION_BUCKET', 'datalab-operation-output-bucket')
+
+# Datalab Archive
+ARCHIVE_API = os.getenv('ARCHIVE_API', 'https://datalab-archive.photonranch.org')
```

Comment:
Let's call this ARCHIVE_API_ROOT so that's consistent with other OCS applications